go.mod

```diff
@@ -4,24 +4,20 @@ go 1.17
 
 require (
 	github.com/go-logr/logr v1.2.3
+	github.com/kcp-dev/kcp/pkg/apis v0.7.0
```

Comment:
both containerized kcp in openshift-pipelines/placement-service and CPS stable as I understand it are at v0.8.2

it may not have an impact on your api consumption based on what I see in your yaml, but conversely, it might be good to stay in sync with the test envs

Reply:
@gabemontero good point. I didn't check it, thinking that the correct version was added automatically. Thanks for pointing out.
Thank you, @fgiloux